config/config.go

```diff
@@ -1117,6 +1117,7 @@ type EC2SDConfig struct {
 	AccessKey       string         `yaml:"access_key,omitempty"`
 	SecretKey       Secret         `yaml:"secret_key,omitempty"`
 	Profile         string         `yaml:"profile,omitempty"`
+	Rolearn         string         `yaml:"rolearn,omitempty"`
```

Comment:
The attribute should be called `RoleARN` to follow the golang naming conventions. For readability, I guess the yaml field should be `role_arn`.